src/context/TransactionContext.jsx

```diff
@@ -63,7 +86,7 @@ export const TransactionsProvider = ({ children }) => {
 
             if (accounts.length) {
                 setCurrentAccount(accounts[0]);
-
+                getAllPuppies();
                 getAllTransactions();
```

Comment:
```suggestion
                await getAllPuppies();
                await getAllTransactions();
```

Since those two functions will return a `Promise` type, suggest adding an `await` keyword before them.